### ISSUE TITLE
Relative wildcard address does not match in sub-projects

### ISSUE THREAD

**Issue by NiklasRosenstein** [kraken-build/kraken#27]
When selecting tasks to run on the command-line, specifying a bare task _name_ matches all tasks with that name in the focus project and all the sub projects. As an example, assuming the following projects and tasks:

```
: (root project)
  :a (sub project) (focus)
    :a:b (sub project)
      :a:b:task (task) 
    :a:task (task)
  :c (sub project)
    :c:task (task)
  :task (task)
```

Running `krakenw query task` with `:a` as the focus project will select the tasks `:a:b:task` and `:a:task`. This is done by transforming the selector from `task` into `.:**:task`.

However, `krakenw query tas*` will only match the task `:a:task`. I would suggest to also transform this selector from `tas*` into `.:**:tas*` automatically as I believe that behavior would be less surprising.

If we do that, the old semantics for matching `tas*` but only in the current project requires that one explicitly specifies the current project, e.g. `.:tas*`.

**Comment by tetigi** [comment]
I think this seems reasonable, and FWIW I don't think many people use this feature (at least, I have never seen it used in the wild - typically just the `task` version that changes to `:**:task`.

**Comment by NiklasRosenstein** [comment]
Implemented in 5d3499c
